src/olympia/versions/views.py

```diff
@@ -58,9 +58,11 @@ def _fetch():
         qs = _version_list_qs(addon)
         return list(qs.values_list('version', flat=True))
 
-    ids = cache_get_or_set(
+    cache_key = make_key(
         u'version-detail:{}:{}'.format(addon.id, version_num),
-        _fetch)
+        normalize=True)
```

Comment:
Alternatively could have just dropped the caching like your TODO said :)

Reply:
One thing at a time :laughing: